docs/topics/cache.txt

```diff
@@ -1123,6 +1135,25 @@ instance, to do this for the ``locmem`` backend, put this code in a module::
 ...and use the dotted Python path to this class in the
 :setting:`BACKEND <CACHES-BACKEND>` portion of your :setting:`CACHES` setting.
 
+.. _asynchronous_support:
+
+Asynchronous support
+====================
+
+.. versionadded:: 4.0
+
+Django has developing support for asynchronous cache backends, but does not
+yet support asynchronous caching. It will be coming in a future release.
+
+``django.core.cache.backends.base.BaseCache`` has async variants of
+:ref:`all base methods <cache-basic-interface>`. To prevent confusion between
+synchronous and asynchronous naming, all asynchronous methods are prefixed
+with ``a``. By default, the arguments for both variants are the same::
```

Comment:
Maybe: 

```suggestion
``django.core.cache.backends.base.BaseCache`` has async variants of :ref:`all
base methods <cache-basic-interface>`. By convention, the asynchronous versions
of all methods are prefixed with ``a``. By default, the arguments for both
variants are the same::
```

Is it really to _prevent confusion_ …? — or is it about _awaitables_ vs regular functions, and then naming conventions on top of that? 🤔 (Not sure)

(In any case, is the wrapping quite right?)

Reply:
I'm 👍 with this. I didn't realize you could wrap the `:ref:` on multiple lines